app/lib/actions/katello/host/destroy.rb

```diff
@@ -33,6 +33,7 @@ def plan(host, options = {})
 
             host.get_status(::Katello::ErrataStatus).destroy
             host.get_status(::Katello::SubscriptionStatus).destroy
+            host.reload
```

Comment:
The reload here https://github.com/Katello/katello/pull/5831/files#diff-50713e26c68a3af05393d936d0f19328R12 is unrelated to this one further down? Figured I'd ask since they are in the same rough area of register/unregister.